pkg/terra/median_contract.go

```diff
@@ -18,9 +18,7 @@ import (
 	"github.com/smartcontractkit/libocr/offchainreporting2/types"
 )
 
-const (
-	BlockRate = 5 // 1 block/5 seconds
-)
+const BlockRateSeconds = 8 // 1 block/8 seconds
```

Comment:
Why don't we extract this to the config, so it's a little less static than this?

Bombay-12 testnet looks to produce blocks every ~6s by my investigation.